plugin/trino-spooling-filesystem/src/main/java/io/trino/spooling/filesystem/FileSystemSpoolingConfig.java

```diff
@@ -91,7 +91,7 @@ public Layout getLayout()
     }
 
     @Config("fs.layout")
-    @ConfigDescription("Spooled segments filesystem layout")
+    @ConfigDescription("Spooling segments file system layout")
```

Comment:
The commit message is not consistent with the change itself. Please undo this and extract to a separate change

Reply:
Pulled apart